ichnaea/locate/provider.py

```diff
@@ -1,8 +1,9 @@
+import json
```

Comment:
Can you import and use simplejson for this? In Python 2.6 the built-in json module is a slow pure-Python implementation.